assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/components/table/Table.tsx

```diff
@@ -23,20 +24,34 @@ const Table: React.FC<TableProps> = ({ prices }) => {
 	const footerRowGenerator = useFooterRowGenerator();
 	const headerRowGenerator = useHeaderRowGenerator();
 
-	const bodyRows = prices.map((price, index) => bodyRowGenerator({ index, price }));
-	const footerRow = footerRowGenerator({ formatAmount, reverseCalculate, toggleCalcDir });
-	const headerRow = headerRowGenerator({ signB4 });
+	const bodyRows = useMemo(() => prices.map((price, index) => bodyRowGenerator({ index, price })), [
+		bodyRowGenerator,
+		prices,
+	]);
+
+	const footerRows = useMemo(() => {
+		const footerRow = footerRowGenerator({ formatAmount, reverseCalculate, toggleCalcDir });
+		return [footerRow];
+	}, [footerRowGenerator, formatAmount, reverseCalculate, toggleCalcDir]);
+
+	const headerRows = useMemo(() => {
+		const headerRow = headerRowGenerator({ signB4 });
+		return [headerRow];
+	}, [headerRowGenerator, signB4]);
+
+	const className = useMemoStringify({ tableClassName: 'ee-ticket-price-calculator' });
+	const metaData = useMemoStringify({
+		tableId: 'ticket-price-calculator-table',
+		tableCaption: __('Ticket Price Calculator'),
+	});
 
 	return (
 		<ResponsiveTable
 			bodyRows={bodyRows}
-			className={{ tableClassName: 'ee-ticket-price-calculator' }}
-			footerRows={[footerRow]}
-			headerRows={[headerRow]}
-			metaData={{
-				tableId: 'ticket-price-calculator-table',
-				tableCaption: __('Ticket Price Calculator'),
-			}}
```

Comment:
Passing object literals as props are a big reason for rerenders

Reply:
ahhhh... ya that makes sense... that's my bad

Reply:
I like to extract this to a variable and reference here after that.